### PR TITLE
Add typespec

### PR DESCRIPTION
**Pull request by yordis**

### CODE REVIEW

**Comment by yordis** [comment]
@TehShrike can you give me a CR 🙏

File: package.json [hunk 1]
```diff
@@ -1,8 +1,9 @@
 {
   "name": "is-mergeable-object",
-  "version": "1.1.0",
+  "version": "1.1.1",
```
Comment (TehShrike) [inline package.json]:
```suggestion
  "version": "1.1.0",
```

My deploy script does this for me using `npm version`

**Comment by TehShrike** [comment]
I don't mind this, but if https://github.com/TehShrike/deepmerge/issues/152 goes forward, this package should cease to matter 😅

**Comment by yordis** [comment]
@TehShrike I am actually using this package for some refactoring thou https://github.com/yordis/webpack-chain/blob/e80fae6563fb84426a688258684f05ea20a358dc/src/ChainedMap.ts#L69 which is the reason why I ended up here 😃 

`deepmerge` does not deal with this case

**Comment by TehShrike** [comment]
Interesting, fair enough.

**Comment by TehShrike** [comment]
Published as 1.1.1

**Pull request merged**